shipcat_cli/src/apply.rs

```
@@ -179,21 +203,17 @@ pub fn apply(svc: &str,
                 info!("successfully applied {} (without waiting)", ui.name);
             }
             else {
-                match kube::await_rollout_status(&mf) {
-                    Ok(success) => {
-                        if success {
-                            info!("successfully rolled out {}", &ui.name);
-                            // TODO: write shipcatmanifest .status here...
-                            webhooks::apply_event(UpgradeState::Completed, &ui, &region, &conf);
-                        } else {
-                            let _ = kube::debug_rollout_status(&mf);
-                            let _ = kube::debug(&mf);
-                            // TODO: collect these for shipcatmanifest .status
-                            warn!("failed to roll out {}", &ui.name);
-                            // TODO: write shipcatmanifest .status here...
-                            webhooks::apply_event(UpgradeState::Failed, &ui, &region, &conf);
-                            return Err(ErrorKind::UpgradeTimeout(mf.name.clone(), mf.estimate_wait_time()).into());
-                        }
+                match kubectl::await_rollout_status(&mf) {
+                    Ok(true) => {
+                        info!("successfully rolled out {}", &ui.name);
+                        webhooks::apply_event(UpgradeState::Completed, &ui, &region, &conf);
+                    },
+                    Ok(false) => {
+                        let _ = kubectl::debug_rollout_status(&mf);
```

What does this do? Is it just printing out a log message?

Also, why are we discarding the result?

---

It prints to stdout speculatively. Don't want to propagate any errors about it.
Was considering capturing some of the information and putting it on the .status object, but that's a different discussion.